website/siteConfig.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const repoUrl = 'https://github.com/clay/amphora';
+
+const siteConfig = {
+  title: 'Amphora', // Title for your website.
+  tagline: 'A new way to organize, edit, and deliver the web, one component at a time',
+  url: 'https://clay.github.io/', // Your website URL
+  baseUrl: '/amphora/', // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
+
+  // Used for publishing and more
+  projectName: 'Amphora',
+  organizationName: 'amphoraplatform',
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: 'intro', label: 'Introduction' },
+    { href: repoUrl, label: 'Github' }
+  ],
+
+  /* path to images for header/footer */
+  headerIcon: 'img/amphora-logo.svg',
+  footerIcon: '',
+  favicon: '',
+
+  /* Colors for website */
+  colors: {
+    primaryColor: '#99D3DF',
+    secondaryColor: '#99D3DF',
+  },
+
+  /* Custom fonts for website */
+  /*
+  fonts: {
+    myFont: [
+      "Times New Roman",
+      "Serif"
+    ],
+    myOtherFont: [
+      "-apple-system",
+      "system-ui"
+    ]
+  },
+  */
+
+  // This copyright info is used in /core/Footer.js and blog RSS/Atom feeds.
+  copyright: `Copyright © ${new Date().getFullYear()} New York Media`,
+
+  highlight: {
+    // Highlight.js theme to use for syntax highlighting in code blocks.
+    theme: 'default',
+  },
+
+  // Add custom scripts here that would be placed in <script> tags.
+  scripts: ['https://buttons.github.io/buttons.js'],
+
+  docsSideNavCollapsible: true,
+  // On page navigation for the current documentation page.
+  onPageNav: 'separate',
+  // No .html extensions for paths.
+  cleanUrl: true,
+
+  // Open Graph and Twitter card images.
+  ogImage: '',
+  twitterImage: '',
+
+  // Show documentation's last contributor's name.
+  // enableUpdateBy: true,
+
+  // Show documentation's last update time.
+  // enableUpdateTime: true,
+
+  // You may provide arbitrary config keys to be used as needed by your
+  // template. For example, if you need your repo's URL...
+  //   repoUrl: 'https://github.com/facebook/test-site',
```

Comment:
Do you think it is possible to remove the commented code? Or do you think is a better idea to leave it as it is?

Reply:
We can delete it. We will have a clean config file